libutils/string_lib.c

```diff
@@ -77,7 +77,7 @@ unsigned int StringHash(const char *str, unsigned int seed, unsigned int max)
     h ^= (h >> 11);
     h += (h << 15);
 
-    return (h & (max - 1));
+    return (h % max);
```

Comment:
But `CF_HASHTABLESIZE` is 8192, which is a valid power of two, and `&` should work fine and the results of `StringHash()` should be distributed perfectly.

I had a quick look, I couldn't find offending code. Can you remind me how you faced the issue? Was it some specific policy?